lib/gems/pending/appliance_console/cli.rb

```diff
@@ -322,6 +329,25 @@ def extauth_opts
       extauthopts.update_configuration(extauthopts_hash)
     end
 
+    def set_server_state
+      service = LinuxAdmin::Service.new("evmserverd")
+      service_running = service.running?
+      case options[:server]
```

Comment:
I think this will be a string at this point, so the `when` statements. Should also be strings.